CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+Thanks for contributing. You rock!
+
+# Submitting an Issue
+
+All issues are tracked as part of the [cesium-ion-community](https://github.com/AnalyticalGraphicsInc/cesium-ion-community) repository.
+
+If you think you've found a bug, first search the [issues](https://github.com/AnalyticalGraphicsInc/cesium-ion-community/issues) there. If an issue already exists, please add a comment expressing your interest and any additional information. This helps us prioritize issue. If you are unable to find an existing issue, please create a new one with as much information needed for us to reproduce the problem or implement a feature.
+
+# Getting Started Contributing
+
+Everyone is welcome to contribute!
+
+See the [Developer Guide](Documentation/Contributors/DeveloperGuide/README.md) for how to set up your system for development.
+
+Always feel free to introduce yourself on the [issue tracker](https://github.com/AnalyticalGraphicsInc/cesium-ion-community/issues) to brainstorm ideas and ask for guidance.
+
+# Opening a Pull Request
+
+We love pull requests. We strive to promptly review them, provide feedback, and merge. Interest in Cesium is at an all-time high so the core team is busy. Following the tips in this guide will help your pull request get merged quickly.
+
+If you plan to make a major change, please start or comment on an [issue](https://github.com/AnalyticalGraphicsInc/cesium-ion-community/issues) about it first. Pull requests for small features and bug fixes can generally just be opened without discussion on the forum.
+
+## Contributor License Agreement (CLA)
+
+Before we can merge a pull request, we require a signed Contributor License Agreement. There is a CLA for:
+
+* [individuals](Documentation/Contributors/CLAs/individual-cla-agi-v1.0.txt) and
+* [corporations](Documentation/Contributors/CLAs/corporate-cla-agi-v1.0.txt).
```

Comment:
This file does not exist.

Reply:
Whoops, meant to update this to point to the cesium repository.  Thanks!

Reply:
Make sure this is linking to the contributing.md file, or if it's copying hte link, to copy the new Google forms.